### ISSUE TITLE
Comparison operators

### ISSUE THREAD

**Issue by cooper-david-a** [mgreminger/EngineeringPaper.xyz#284]
Are comparison operators supported?
If not, can they be?

![image](https://github.com/user-attachments/assets/dbb017fb-19ef-48e4-b189-fc35e011002b)

Piecewise expressions are a good workaround.
![image](https://github.com/user-attachments/assets/26b0f1f7-22ab-44a2-82ae-44539f73a11c)

**Comment by mgreminger** [comment]
Yes, inequalities are only supported in piecewise functions at the time being. You can use undefined variables to give a little more informative output as shown below:

![image](https://github.com/user-attachments/assets/ad1f8d7a-85cb-443e-9782-8e4dfaf24743)
